x-pack/plugins/cases/public/components/custom_fields/translations.ts

```diff
@@ -59,6 +59,10 @@ export const FIELD_OPTIONS = i18n.translate('xpack.cases.customFields.fieldOptio
   defaultMessage: 'Options',
 });
 
+export const DEFAULT_VALUE = i18n.translate('xpack.cases.customFields.defaultValue', {
+  defaultMessage: 'Default Value',
```

Comment:
To my understanding, the writing guidance dictates that the first letter of the second world should be lowercase, `Value` -> `value`. @lcawl is this correct?